test/extensions/filters/http/common/fuzz/filter_fuzz_test.cc

```diff
@@ -13,9 +13,9 @@ namespace Extensions {
 namespace HttpFilters {
 
 DEFINE_PROTO_FUZZER(const test::extensions::filters::http::FilterFuzzTestCase& input) {
-  static PostProcessorRegistration reg = {[](test::extensions::filters::http::FilterFuzzTestCase*
-                                                 input,
-                                             unsigned int seed) {
+  static PostProcessorRegistration reg __attribute__((
```

Comment:
use `ABSL_ATTRIBUTE_UNUSED` so it won't affect platforms doesn't support __attribute__? (likely MSVC)

Reply:
done.